src/actions/undoControlPanel/transactions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Transactions actions.
+ * @module actions/undoControlPanel/transactions
```

Comment:
Better keep "undoControlPanel" out of the naming since that is an implementation detail. The actions and reducers are more generic so the file should be: "src/actions/transactions/transactions.js".